Doc/reference/compound_stmts.rst

```diff
@@ -680,8 +680,13 @@ can be used to create instance variables with different implementation details.
 
 .. seealso::
 
-   :pep:`3115` - Metaclasses in Python 3
+   :pep:`3115` - Metaclasses in Python 3000
+      The proposal that introduced the current syntax for declaring metaclasses,
+      and the semantics for how classes with metaclasses are constructed.
+
    :pep:`3129` - Class Decorators
+      The proposal that added class decorators, as an extension to
```

Comment:
Perhaps:

The proposal that added class decorators. Function and method decorators were introduced earlier in :pep:`318`.